WordPress/Classes/ViewRelated/Stats/Shared Views/StatsTotalRow.swift

```diff
@@ -199,21 +201,21 @@ private extension StatsTotalRow {
             return
         }
 
+        let imageSize = rowData.statSection?.imageSize ?? StatSection.defaultImageSize
+        imageWidthConstraint.constant = imageSize
+
         imageView.isHidden = !hasIcon
 
         if let icon = rowData.icon {
-            imageWidthConstraint.constant = Constants.defaultImageSize
             imageView.image = icon
         }
 
         if let iconURL = rowData.socialIconURL {
-            imageWidthConstraint.constant = Constants.socialImageSize
             downloadImageFrom(iconURL)
         }
 
         if let iconURL = rowData.userIconURL {
-            imageWidthConstraint.constant = Constants.userImageSize
-            imageView.layer.cornerRadius = Constants.userImageSize * 0.5
+            imageView.layer.cornerRadius = imageSize * 0.5
```

Comment:
I know this code was in place previously, but it occurred to me that some might not understand the intent of this message.

If we're making the image view circular, it _might_ be more clear to set the path for a mask via one of the following methods:

- `init(arcCenter:radius:startAngle:endAngle:clockwise:)`
- `init(ovalIn:)`

I don't feel strongly about this, but wanted to share.